main.go

```diff
@@ -68,6 +69,10 @@ var (
 )
 
 const (
+	reservedPath  = "/minio"
+	profilingPath = reservedPath + "/pprof"
+
+	// TODO: replace /.prometheus with /minio/prometheus/metrics
```

Comment:
This is chosen on purpose since there can never be buckets starting with a `.`  - so you can use `.pprof`